src/shogun/modelselection/ParameterCombination.h

```diff
@@ -215,6 +215,25 @@ friend class CModelSelectionParameters;
 		return "ParameterCombination";
 	}
 
+	/** helper method used to specialize a base class instance
+	 *
+	 * @param param_combination its dynamic type must be CParameterCombination
+	 */
+	static CParameterCombination* obtain_from_generic(
+			CSGObject* param_combination)
+	{
+		if (param_combination)
+		{
+			CParameterCombination* casted = dynamic_cast<CParameterCombination*>(param_combination);
+			REQUIRE(casted, "CParameterCombination::obtain_from_generic(): Error, provided object"
```

Comment:
Can remove the "CParameterCombination::obtain_from_generic(): ".
We add that automatically now